metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -1085,3 +1099,13 @@ def _get_ge_dataset(
                 logger.debug(f"Setting table name to be {batch._table}")
 
         return batch
+
+
+# More dialect specific types to ignore can be added here
+# Stringified types are used to avoid dialect specific import errors
+@lru_cache(maxsize=1)
+def _get_column_types_to_ignore(dialect_name: str) -> List[str]:
```

Comment:
let's change this to _should_ignore_column(dialect_name, type_str) -> bool

in the future we'll run into weird `struct<...>` and things, and so letting this function handle it is easier

Reply:
sure. One point to note is the type_str here is not native type but sqlalchemy type. This is description of `type` before stringification - 

```
* ``type`` - the type of this column; an instance of
          :class:`~sqlalchemy.types.TypeEngine`
```